src/doc/style-guide/src/expressions.md

```diff
@@ -663,7 +663,8 @@ never use a block (unless the block is empty).
 
 If the right-hand side consists of multiple statements, or has line comments,
 or the start of the line does not fit on the same line as the left-hand side,
```

Comment:
One alternative I considered here was something to the effect of "or is a single macro call". However, I opted against that because that could be interpreted as prescribing a change in behavior (wherein devs/tools would need to newly add blocks around existing arms (e.g. `foo => bar!()` --> `foo => { bar!() }` [linebreaks elided for simplicity of example]).

That would have resulted in a breaking change on the rustfmt side